.rubocop.yml

```diff
@@ -0,0 +1,34 @@
+AllCops:
```

Comment:
– Need to add rubocop usage to CI
– I have a lot of alerts from rubocop locally

Reply:
I have setup rubocop on TravisCI - got no warnings